swift/ql/test/query-tests/Security/CWE-312/testUserDefaults.swift

```diff
@@ -0,0 +1,70 @@
+
+// --- stubs ---
+
+class NSObject
+{
+}
+
+class UserDefaults : NSObject
+{
+    class var standard: UserDefaults {
+        return UserDefaults()
+    }
+
+	func set(_ value: Any?, forKey key: String) {}
```

Comment:
And this:
```
func set(_ value: Any?, forKey defaultName: String) {}
```
(https://developer.apple.com/documentation/foundation/userdefaults/1414067-set)

Reply:
Ditto.